activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -354,17 +354,24 @@ def group!(*args) # :nodoc:
       self
     end
 
-    # Allows to specify an order attribute:
+    # Order can be used to sort the results of a query.
     #
+    # #order accepts arguments in one of several formats.
+    #
+    # === symbols
+    # The symbol represents the name of the column you want to order the results by.
```

Comment:
I think the convention is to have a new line before and after all code examples:
```suggestion
    # The symbol represents the name of the column you want to order the results by.
    #
```